.github/workflows/release.yaml

```diff
@@ -163,7 +164,27 @@ jobs:
         id: get-version
         run: |
           ./radius/.github/scripts/release-get-version.sh ${{ steps.get-supported-versions.outputs.result }} radius
+      - name: Check if release branch exists
+        id: release-branch-exists
```

Comment:
Could line179 `context.ref == 'refs/heads/main' && context.eventName == 'push'` be moved here?

Reply:
I don't think so, because we need `steps.release-branch-exists.outputs.result` to be `'false'` for the rest of the workflow run to make sure the other steps are skipped